### ISSUE TITLE
DOES MINET HAVE 32 BIT ISSUE?

### ISSUE THREAD

**Issue by bettytwin** [NiclasOlofsson/MiNET#289]

**Comment by NiclasOlofsson** [comment]
No, not that i know of. But it may very well be that .NET have issues with it.

**Comment by bettytwin** [comment]
what net ? internet?

**Comment by NiclasOlofsson** [comment]
Well, internet is driven by 32 bits, so they would totally have that problem. But I mean Microsoft .NET. You know, that crap that Bill Gates forced on us back when he was really evil. But tell me, what is that thing with netflix. Why do they need pocketmine? Are they going to partner with Microsoft and Mojang now? Is the Minecraft movie going to be lanched on Netflix using PMMP?
